demisto_sdk/commands/create_id_set/tests/create_id_set_test.py

```diff
@@ -299,3 +299,97 @@ def test_do_not_modify_specific_brand(repo):
         assert playbook1['command_to_integration']['generic-command'] == ['MainInteg', 'SecondaryInteg']
         assert playbook2['command_to_integration']['generic-command'] == 'MainInteg'
         assert playbook2['command_to_integration']['no-integration'] == ''
+
+    @staticmethod
+    def test_generic_command_that_does_not_use_a_specific_brand(repo):
+        """
+        Given
+        - A playbook with a generic command (send-notification, etc.) that does not use a specific brand
+
+        When
+        - Updating the commands_to_integrations fields in playbooks
+
+        Then
+        - Do not update 'command_to_integration' fields with any other brands
+
+        """
+        integrations = [
+            {
+                'Slack': OrderedDict([
+                    ('name', 'Slack'),
+                    ('commands', ['send-notification']),
+                ])
+            },
+            {
+                'Syslog': OrderedDict([
+                    ('name', 'Syslog'),
+                    ('commands', ['send-notification']),
+                ])
+            },
+        ]
+        playbooks = [
+            {
+                'Playbook': OrderedDict([
+                    ('name', 'Playbook'),
+                    ('command_to_integration', {
+                        'send-notification': '',
+                    }),
+                ]),
+            }
+         ]
+
+        id_set_creator = IDSetCreator(print_logs=False)
+        id_set_creator.id_set["integrations"] = integrations
+        id_set_creator.id_set["playbooks"] = playbooks
+
+        id_set_creator.add_command_to_implementing_integrations_mapping()
+
+        playbook = id_set_creator.id_set['playbooks'][0]['Playbook']
+        assert playbook['command_to_integration']['send-notification'] == ''
+
+    @staticmethod
+    def test_generic_command_that_uses_a_specific_brand(repo):
+        """
+        Given
+        - A playbook with a generic command (send-notification, etc.) that uses a specific brand
+
+        When
+        - Updating the commands_to_integrations fields in playbooks
+
+        Then
+        - Do not update 'command_to_integration' fields with any other brands
```

Comment:
We do need to update in this case aren't we?

Reply:
The reason we do not need to update, is because `command_to_integration` is already updated by 
![Screen Shot 2021-07-25 at 12 40 12](https://user-images.githubusercontent.com/78307768/126894748-7337c127-69a0-4c68-9467-30cc0ae5b911.png)
from this file
demisto_sdk/commands/common/update_id_set.py.

the purpose of `id_set_creator.add_command_to_implementing_integrations_mapping()` function is to update `command_to_integration` in case of a **_non-generic-command_** that is **_not_** depend on a specific brand